### PR TITLE
add token storage and logout

### PR DESCRIPTION
**Pull request by downdawn**
#44

### CODE REVIEW

File: backend/app/common/jwt.py [hunk 1]
```diff
@@ -42,28 +43,34 @@ def password_verify(plain_password: str, hashed_password: str) -> bool:
...
+    :param sub: The subject/userid of the JWT
     :param data: Data transferred to the token
     :param expires_delta: Increased expiry time
     :return:
     """
     if expires_delta:
         expires = datetime.utcnow() + expires_delta
+        expire_seconds = expires_delta.total_seconds()
     else:
         expires = datetime.utcnow() + timedelta(settings.TOKEN_EXPIRE_MINUTES)
```
Comment (wu-clan) [inline backend/app/common/jwt.py:59]: Hi, @downdawn 

We have a bug here that needs to be fixed, the parameter in timedelta needs to be specified minutes

**Review by wu-clan — CHANGES_REQUESTED** [review]

File: backend/app/common/jwt.py (hunk 1 above)
Comment (wu-clan) [reply backend/app/common/jwt.py:59]: timedelta(minutes=settings.TOKEN_EXPIRE_MINUTES)

**Comment by wu-clan** [comment]
LGTM if update it

File: backend/app/common/jwt.py (hunk 1 above)
Comment (wu-clan) [reply backend/app/common/jwt.py:59]: Or we can change it to be timed in seconds

**Comment by wu-clan** [comment]
Hi, @downdawn 

I have a new PR #59 that needs it merged, please update and push

File: backend/app/common/jwt.py (hunk 1 above)
Comment (downdawn) [reply backend/app/common/jwt.py:59]:
```
TOKEN_EXPIRE_SECONDS: int = 60 * 24 * 1 * 60  # token 时效 60 * 24 * 1 * 60 = 1 天
```
```
    if expires_delta:
        expires = datetime.utcnow() + expires_delta
        expire_seconds = expires_delta.total_seconds()
    else:
        expires = datetime.utcnow() + timedelta(seconds=settings.TOKEN_EXPIRE_MINUTES)
        expire_seconds = settings.TOKEN_EXPIRE_SECONDS
```

Comment (wu-clan) [reply backend/app/common/jwt.py:59]: Great!

Comment (downdawn) [reply backend/app/common/jwt.py]: WIP

**Review by wu-clan — APPROVED** [review]

**Comment by wu-clan** [comment]
LGTM.

**Pull request merged**